docs/wiki/NuGet-packages.md

```diff
@@ -13,6 +13,31 @@ This package is primarily intended as a method for rapidly shipping hotfixes to
 Note: this package is intended as a replacement for Microsoft.Net.Compilers (which is a Windows-only package) and Microsoft.NETCore.Compilers. Those packages are now deprecated and will be deleted in the future.
 
 ## Versioning
+Starting on Roslyn versions `5.0` and above, Roslyn NuGet packages are shipped with the .NET SDK instead of VS.  New Roslyn packages will be published on every SDK preview and GA release.
+   
+Prior to this change, a Roslyn package was matched exactly to a VS version, and did not exactly match SDK versions.  Now, Roslyn packages will exactly align with an SDK version, but will no longer exactly match a VS version.  While generally the SDK releases ship alongside new VS versions, there may be drift in the exact commit of Roslyn when compared to VS (especially for prereleases).
+
+### Version Schema
+Generally Roslyn package versions are in the form A.B.C-D.  With the new publishing mode this will be represented as
+* A = Associated with the current VS Major version (`5` = VS 18, same as before).
+* B = The current VS Minor version (same as before)
+* C = Our own patch version if we need to service the package for any reason, generally `0`.
+* D = The VS prerelease version `-` VMR build version (if any).
+
+We still use VS versions in our version scheme as Roslyn currently branches for releases with VS in mind.
+
+What this actually looks like:
+
+| SDK Version | 11.0.100-preview.4 | 11.0.100-rc.2 | 11.0.100 | 11.0.200 |
+|---|---|---|---|---|
+| Roslyn version | 5.6.0-2.25465.10 | 5.8.0-3.25501.82 | 5.9.0 | 5.10.0 |
+
+SDK GA releases align with VS Minor versions.  So going from `11.0.100` to `11.0.200` means a new VS Minor version, hence incrementing the Roslyn package minor version.
+
+However, SDK prereleases do not always correspond to new VS Minor or VS preview versions.  For example, while `11.0.100-preview.4` might produce Roslyn `5.6.0-2-25465.101`, the subsequent `11.0.100-preview.5` could produce Roslyn `5.6.0-2-25503.114` without any change to the VS minor or preview version.  In these cases, we need to include the VMR build number to differentiate between packages.  This typically occurs with early SDK versions or during extended VS preview cycles (such as preview 1 for a new VS major version).
```

Comment:
alternative here is setting our prerelease label to the SDK prerelease and manually incrementing for SDK releases.  But that seems a bit more complicated due to us branching for VS, and not the SDK.